specification/source/cdl.rst

```diff
@@ -1688,118 +1688,75 @@ Therefore, tools that process CDL can infer the following information:
 
 .. _sec_tag_pro:
 
-Tagged Properties
+Semantic Information
 .................
 
-The buildings industry uses different tagging schemes such as
-Brick (https://brickschema.org/) and Haystack (https://project-haystack.org/).
-CDL allows, but does not require, use of the Brick or Haystack tagging scheme.
+The buildings industry uses different metadata schemes such as
+`Brick <https://brickschema.org/>`_, `Project Haystack <https://project-haystack.org/>`_ and `ASHRAE 223p <https://www.ashrae.org/about/news/2018/ashrae-s-bacnet-committee-project-haystack-and-brick-schema-collaborating-to-provide-unified-data-semantic-modeling-solution>`_. CDL allows, but does not require, use of the Brick or Haystack or any other tagging scheme.
 
-CDL allows to add tags to declarations that instantiate
+* Semantic information will be included within the CDL ``annotation`` keyword, using the ``__semantic`` annotation. Every element in CDL can have an annotation and correspondingly, semantic information
+ * Inputs
+ * Outputs
+ * Parameters
+ * CDL blocks: both elementary blocks (:numref:`sec_ele_blo`), and composite blocks (:numref:`sec_com_blo`)
+ * Packages
 
-* elementary blocks (:numref:`sec_ele_blo`), and
-* composite blocks (:numref:`sec_com_blo`).
+* The top-level CDL block declaration shall house the `metadataLanguageDefinition` within the ``__semantic`` annotation. ``metadataLanguageDefinition`` will be used to define the different types of tags that are used throughout the CDL block and relevant information about the same. It will follow the following syntax:
 
-[We currently do not see a use case that would require
-adding a tag to a ``parameter`` declaration.]
+.. code-block:: modelica
 
+   annotation (__semantic(metadataLanguageDefinition="<metadataLanguage> [version] [format]" "<description or URL>"))
```

Comment:
I am not opposed to adding the optional ``[format]`` but isn't the format always implied by `metadataLanguage`?

Reply:
Not always - For example, Haystack alone has multiple formats: JSON, JSON+LD, Zinc, Turtle etc. (https://project-haystack.org/example)